### PR TITLE
Add nixd

### PR DESCRIPTION
**Pull request by Ruixi-rebirth**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -133,6 +133,7 @@
 * [namaka](https://github.com/nix-community/namaka) - Snapshot testing for Nix based on haumea.
 * [nil](https://github.com/oxalica/nil) - NIx Language server, an incremental analysis assistent for writing in Nix.
 * [niv](https://github.com/nmattia/niv/) - Easy dependency management for Nix projects with package pinning.
+* [nixd](https://github.com/nix-community/nixd) - Nix language server, based on nix libraries.
```
Comment (cyntheticfox) [inline README.md:136]: Capitalize "nix" please, and move the entry below "nix-update" to maintain alphabetical sorting (I admittedly just use (neo)vim's sort function).

**Review by cyntheticfox — CHANGES_REQUESTED** [review]
The description itself is alright, though admittedly brief and kinda underselling the _accuracy_ given by linking with Nix libs, but it works. Please fix the minor grammar request and move the entry as explained in the line comment, and then we'll be good.

File: README.md (hunk 1 above)
Comment (Ruixi-rebirth) [reply README.md]: done

**Review by cyntheticfox — APPROVED** [review]
Looks good to me

**Pull request merged**